dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -78,17 +93,39 @@ public static void dropAndRecreateNoTestData(DropwizardTestSupport<DockstoreWebs
      * @throws Exception
      */
     public static void dropAndCreateWithTestData(DropwizardTestSupport<DockstoreWebserviceConfiguration> support, boolean isNewApplication) throws Exception {
+        dropAndCreateWithTestData(support, isNewApplication, CONFIDENTIAL_CONFIG_PATH);
+    }
+
+    public static void dropAndCreateWithTestData(DropwizardTestSupport<DockstoreWebserviceConfiguration> support, boolean isNewApplication, String dropwizardConfigurationFile) throws Exception {
         LOG.info("Dropping and Recreating the database with non-confidential test data");
         Application<DockstoreWebserviceConfiguration> application;
         if (isNewApplication) {
             application = support.newApplication();
         } else {
             application= support.getApplication();
         }
-        application.run("db", "drop-all", "--confirm-delete-everything", CONFIDENTIAL_CONFIG_PATH);
+        application.run("db", "drop-all", "--confirm-delete-everything", dropwizardConfigurationFile);
 
         List<String> migrationList = Arrays.asList("1.3.0.generated", "1.3.1.consistency", "test", "1.4.0", "1.5.0", "test_1.5.0", "1.6.0", "1.7.0");
-        runMigration(migrationList, application, CONFIDENTIAL_CONFIG_PATH);
+        runMigration(migrationList, application, dropwizardConfigurationFile);
+    }
+
+    /**
+     * Shared convenience method
+     * @return
+     */
+    public static ApiClient getWebClient(boolean authenticated, String username) {
```

Comment:
Is this being moved from somewhere?
I don't see a corresponding delete anywhere